tests/Dialect/TfheRustBool/IR/ops.mlir

```diff
@@ -32,4 +32,19 @@ module {
     %out = tfhe_rust_bool.and %bsks, %lhs, %rhs: (!bsks, tensor<4x!eb>, tensor<4x!eb>) -> tensor<4x!eb>
     return
   }
+
+  // CHECK-LABEL: func @test_not
+  func.func @test_not(%bsks : !bsks) {
+    %0 = arith.constant 1 : i1
+
+    %e1 = tfhe_rust_bool.create_trivial %bsks, %0 : (!bsks, i1) -> !tfhe_rust_bool.eb
+    %out = tfhe_rust_bool.not %bsks, %e1: (!bsks, !tfhe_rust_bool.eb) -> !tfhe_rust_bool.eb
+    return
+  }
+
+  // CHECK-LABEL: func @test_packed_not
```

Comment:
These tests are not necessary. We only have separate parser tests for the PRs in which we add ops but don't use them for anything in that PR. Since the other tests use packed_not already, they also test the parser.